src/main/java/io/supertokens/webserver/api/totp/CreateOrUpdateTotpDeviceAPI.java

```diff
@@ -4,15 +4,16 @@
 import java.security.NoSuchAlgorithmException;
 
 import com.google.gson.JsonObject;
-
 import io.supertokens.Main;
 import io.supertokens.pluginInterface.RECIPE_ID;
```

Comment:
add tests for all these API changes to make sure that user id mapping is happening